public/assets/main.css

```diff
@@ -1,11 +1,20 @@
-* { font-family: 'Inconsolata', monospace; }
+* {
+  font-family: Lato, sans;
+  word-spacing: 1px;
+}
 
 a {
   color: currentColor;
   text-underline-offset: 0.2em;
 }
+nav a {
+  text-decoration: none;
+}
+nav a.active {
```

Comment:
does this need to be a class? if it's not does the focus not work?

Reply:
No, it was just the/my usual way. `route_anchor == anchor[1..-1] ? a_props[:style][:textDecoration] = 'underline' : ''` would work, too.

Reply:
ok let’s revert it